package.json

```diff
@@ -9,6 +9,7 @@
     "jest-styled-components": "^7.0.3",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
+    "react-hook-form": "^6.11.4",
```

Comment:
So you've installed this, but didn't use it?

Reply:
I used it, but decided to go without and forgot about the package.json. Thanks! :)